### ISSUE TITLE
List most recent public queries

### ISSUE THREAD

**Issue by nichtich** [ColinMaudry/sparql-router#91]
I wondered where to view existing queries but without read-only views (#92) such feature is obviously not a good idea.

**Comment by ColinMaudry** [comment]
Yes, listing existing public queries is part of the plan, but I first want to improve existing features:
- improving table results #8 
- having some sort of graph visualization (even if at the beginning it's just a Turtle serialization) #60
- making the query type (tables/graphs) automatically defined #50 #67 

Could you please be more specific about what's "obviously not a good idea"? Enabling certain queries to be read-only?

Thanks!
